README.md

```diff
@@ -90,6 +90,10 @@ Note: Both operations copy the existing file.
 	$docs = $dms->getByTag('priority', 'important')->First();
 	$link = $doc->getLink();
 
+	// Set default download behavior ('open' or 'download'). 'download' is the system default
+	// Attempt to open the file in the browser
+	DMSDocument_Controller::set_default_download_behavior('open');
```

Comment:
This needs updating to: `$config->update('DMSDocument', 'default_download_behaviour', 'open');`